_posts/2017-05-15-rust-at-two-years.md

```diff
@@ -0,0 +1,356 @@
+
+*Thanks to Jonathan Turner for this RLS summary!*
+
+### [Rust should have 1.0-level crates for essential tasks](https://github.com/rust-lang/rust-roadmap/issues/11), and [Rust should provide easy access to high quality crates](https://github.com/rust-lang/rust-roadmap/issues/9)
+
+The [recent post on the Libz Blitz] details the Library Team's initiative to
+increase the quality of crates for common tasks; that post is excellent so I
+won't repeat it here. I will note that many of the issues that the Libs Team
+is going to create will be great starter issues. For the blitz to be the best
+it can be, the Libs Team is going to need help from the community-- that means
+YOU! :) They're willing to mentor people interested in contributing.
+
+In order to make awesome crates easier to find for particular purposes,
+crates.io now has [categories] for crate authors to better indicate the use
+case of their crate. Crates can also now have CI badges, and
+[more improvements to crates.io's interface] are coming that will help you
+choose the crates that fit your needs.
+
+[recent post on the Libz Blitz]: https://blog.rust-lang.org/2017/05/05/libz-blitz.html
+[categories]: https://crates.io/categories
+[more improvements to crates.io's interface]: https://github.com/rust-lang/rust/issues/41616
+
+### [Rust should be well-equipped for writing robust, high-scale servers](https://github.com/rust-lang/rust-roadmap/issues/10)
+
+TODO: [Futures] and [tokio] enable zero-cost asynchronous programming
+
+[Futures]: https://crates.io/crates/futures
+[tokio]: https://tokio.rs/
+
+### [Rust should integrate easily into large build systems](https://github.com/rust-lang/rust-roadmap/issues/12)
+
+This initiative is mostly in the ideas stage; there's lots of great discussion
+on the tracking issue that has resulted in a few Cargo issues:
+
+* [Support creating a build plan](https://github.com/rust-lang/cargo/issues/3815)
+* [Support declaring external dependencies](https://github.com/rust-lang/cargo/issues/3816)
+
+Keep an eye out for more improvement in this area soon.
+
+### [Rust's community should provide mentoring at all levels](https://github.com/rust-lang/rust-roadmap/issues/13)
+
+There were [RustBridge] events before RustFest Berlin and Rust Belt Rust, and
+there's another planned for the day before RustConf coming up!
+
+[RustBridge]: https://github.com/rust-community/rustbridge
+
+There are many people who have started getting involved with various
+initiatives. I've worked with many people at many places in their Rust journey:
+helping out with the conferences, giving their first conference talks,
+providing feedback on the book, working on crates, contributing to Rust itself,
+and joining teams! While we haven't made as much formal, measurable progress on
+this front, everywhere I turn I see Rustaceans helping other Rustaceans.
+
+## Rust in the future
+
+While Rust is still just a toddler, it's still growing and still going strong.
```

Comment:
Do you have any suggestions? What would be stronger? More exclamation points maybe!!!! I'm feeling at a loss for words after all the rest of the words :)